src/page/browser.js

```diff
@@ -0,0 +1,49 @@
+function isMinimumSupported() {
```

Comment:
Let's add some comments here that explains why this file is `js` only and what it is for exactly

Reply:
done